### PR TITLE
add schema type hints for some go types

### PR DESCRIPTION
**Pull request by ripienaar**
Signed-off-by: R.I.Pienaar <rip@devco.net>

### CODE REVIEW

File: schema_source/jetstream/api/v1/definitions.json [hunk 1]
```diff
@@ -5,6 +5,33 @@
...
   "type": "object",
   "definitions": {
+    "golang_duration_nanos": {
+      "$comment": "nanoseconds depicting a duration in time",
+      "$ref": "#/definitions/golang_int64"
+    },
+    "golang_int": {
+      "$comment": "integer with a dynamic bit size depending on the platform the cluster runs on, can be up to 64bit",
+      "minimum": -9223372036854775808,
+      "maximum": 9223372036854775807
```
Comment (ripienaar) [inline schema_source/jetstream/api/v1/definitions.json:15]: @kozlovic mainly added you to review to double check this is a correct scription of `int`

File: schema_source/jetstream/api/v1/definitions.json [hunk 2]
```diff
@@ -5,6 +5,33 @@
   "description": "Shared definitions for the JetStream API",
   "type": "object",
   "definitions": {
+    "golang_duration_nanos": {
+      "$comment": "nanoseconds depicting a duration in time",
```
Comment (scottf) [inline schema_source/jetstream/api/v1/definitions.json:9]: can you change the comment to "nanoseconds depicting a duration in time, signed xx bit integer"

Comment (ripienaar) [reply schema_source/jetstream/api/v1/definitions.json]: fixed this, also put in some `sed` hacks to fix these numbers :( go would panic on parsing the ones node produce, this really is a mess.

**Review by scottf — APPROVED** [review]
LGTM

File: schema_source/jetstream/api/v1/definitions.json (hunk 1 above)
Comment (kozlovic) [reply schema_source/jetstream/api/v1/definitions.json]: Yes, it is the correct range for `int` on a 64bit platform.

**Review by kozlovic — APPROVED** [review]
LGTM

**Pull request merged**